### DISCUSSION TITLE
MUMPS version

### DISCUSSION THREAD

**Discussion by heinono1** [idaholab/moose#25720] [Q&A Getting Started]
Hi. I just pulled a new distribution of MOOSE to re-build from scratch. Because of our IT Security trolls, I could not pull MUMPS_5.6.1 from the graal.ens-lyon.fr site and so the installer looked for MUMPS at the anl.gov sites. It turns out the anl.gov sites are not in synch with the ens-lyon.fr site and only had version 5.6.0. Easy fix to just change the version in the install python script but I thought it could be worth to point this out if others run into trouble and don't know how to fix it.

**Comment by milljm** [comment]
What is strange to me, is that I only run into this issue when attempting to build PETSc with MUMPS options while on an Apple Silicon machine. I can't explain why Linux and Intel Macs are fine 😕

**Reply by heinono1** [reply]
Haha I am installing on Linux clusters and a Linux workstation (Ubuntu 22.04) - and I get the errors.....
